src/pageLayout/containers/TreeNav.Helpbar.test.tsx

```diff
@@ -2,17 +2,46 @@ import React from 'react'
 import {screen, fireEvent, cleanup, waitFor} from '@testing-library/react'
 import {jest} from '@jest/globals'
 
+// Mocks
```

Comment:
The mock being used in this test was outdated because it required references to `quartzMe`.